ml-agents/mlagents/trainers/policy/tf_policy.py

```diff
@@ -368,6 +370,9 @@ def save_model(self, steps):
         :param steps: The number of steps the model was trained for
         :return:
         """
+        if hvd.rank() != 0:
```

Comment:
So that only one model/the model being trained on process id 0 is saved?

Reply:
yes, that's correct.